kms_keymaker_multiregion_replica/variables.tf

```diff
@@ -14,6 +14,11 @@ variable "sqs_queue_arn" {
   description = "ARN of the SQS queue used as the CloudWatch event target."
 }
 
+variable "alarm_sns_topic_arn" {
+  type        = string
+  description = "ARN of the SNS topic used to send KMS alarms."
+}
+
```

Comment:
If this only shows up in the (currently-inactive) `kms_keymaker_multiregion_replica/sns.tf` file, comment it out for now so that Terraform doesn't complain about unused/unset variables.